writing/milestone-0/OVERVIEW.md

```diff
@@ -0,0 +1,80 @@
+
+In practice this means a single initial machine with cables that can be plugged into one or more "expansion boxes". People can invest in expanding these systems over many years. With each expansion, the system can assist in more activities and bring more smarts to each of them.
+
+We have plans for single-, double-, and quad-accelerator expansion boxes that can be plugged into a variety of machines. The plans will be available in early 2024. The double reuses all the parts of the single, and the quad reuses all the parts of the single and double.
+Who It's For
+Initially, this project will serve hackers who want to build and collaborate with one another on local and intelligent systems. 
+
+### Who is it for? What are some sample situations?
+
+A1. Hackers (working on an open source software/hardware project)
+
+A2. Students (in grad school doing research, with international and multilingual collaborators)
+
+A3. Small and mid-sized businesses (a regional distributor)
+
+
+### What It Is
+
+#### What you get in the mail:
+- Substrate System 0 is an easy-to-assemble AI computer kit for home and work.
+
+#### Once built, it runs:
+- SubstrateOS, an operating system that boots from a USB drive and schedules AI models on internal and external consumer GPUs.
+
+#### Once booted, it can:
+- M0.1. Join in on real-world conversations via your other devices. [A1] [A2] [A3]
+    A1. Planning meeting
+    A2. Lecture
+    A3. Sales meeting, customer call
+
+- M0.2. Take meeting and lecture notes so you can pay full attention to the speakers. [Liv] [A1] [A2] [A3]
+    A1. Meeting minutes, next steps
+    A2. Lecture notes
+    A3. ?
+
+- M0.3. Identify and link to more info on people, events, and cited documents. [Liv] [A1] [A2]
+    A1. ?
+    A2. Important dates for the course; references to other talks, textbooks, papers, textbooks
+
+- M0.4. Keep an at-hand, ever-present record of conversation, tasks, and specific contributions. This captures good ideas as they happen, foster new connections between them, and helps maintain understanding between collaborators. [Matt] [Liv] [A1] [A2] [A3]
+    A1. Feature suggestions, observations, 
+    A2. ?
+    A3. ?
+
+- M0.5. Eliminate language barriers by translating between any number of supported languages in real-time, across speech and text. [A2] [A3]
+    A2. ?
+    A3. ?
+
+- M0.6. Respond only when actively addressed directly or indirectly, so using its name in a sentence does not result in false positives, but people also don't need to keep using its name explicitly to continue a conversation.
+
+- M0.7. Self-assess its reasoning capabilities so it knows what it can do and how well it performs relative to expectations.
+
+- M0.8. Cast wirelessly to nearby displays. [?Jeff] [A1] [A2] [A3]
+    A1. ?
+    A2. ?
+    A3. ?
+
+- M0.9. Help draft comparative analyses of ideas shared in both documents and live interactions. [A1] [A2]
+    A1. ?
+    A2. ?
```

Comment:
A2. [Aspirational] As a graduate student, many of my classes require a final paper to demonstrate what we learned in the course and to create our own novel assessments or proposals of new, related ideas. The documents that support the writing of these papers are a combination of supplemental readings (books, articles), verbal notes and slides from lectures, insights from linking in-course knowledge to other concepts and topics that I've learned before. With a Substrate system, I would like to be able to use the system's knowledge of all of the materials that I'm consuming related to the course, so that as I dynamically build an outline and start filling in my final paper, it can relate this back to the other materials and help me with developing out related ideas/synthesis and citation.